lib/location/method_gnss.c

```diff
@@ -223,13 +231,15 @@ static void method_gnss_agps_request_work_fn(struct k_work *item)
 
 	struct nrf_cloud_rest_agps_result result = {agps_data_buf, sizeof(agps_data_buf), 0};
 
+#if defined(CONFIG_NRF_CLOUD_REST)
```

Comment:
All changes in this file are experimental to get this file compiled without MQTT and REST.
Don't know what we should do but we'd need to handle this configuration somehow so that `CONFIG_LOCATION_METHOD_GNSS_AGPS_EXTERNAL` is handled properly.

Reply:
Changes made and should be fine now.